property_tests/arbitraries/candid/constructed/vec_arb.ts

```diff
@@ -11,23 +10,28 @@ import { Nat16Arb } from '../primitive/nats/nat16_arb';
 import { Nat32Arb } from '../primitive/nats/nat32_arb';
 import { Nat64Arb } from '../primitive/nats/nat64_arb';
 import { PrincipalArb } from '../reference/principal_arb';
-import { Candid, CandidTypeArb } from '..';
+import { Candid } from '..';
 import { BoolArb } from '../primitive/bool';
 import { Float32Arb } from '../primitive/floats/float32_arb';
 import { Float64Arb } from '../primitive/floats/float64_arb';
 import { NullArb } from '../primitive/null';
 import { TextArb } from '../primitive/text';
+import { deepEqual } from 'fast-equals';
 
 const VecInnerArb = <T>(arb: fc.Arbitrary<Candid<T>>) => {
-    return fc.tuple(fc.array(arb), arb).map(
-        ([sample, src]): Candid<T[]> => ({
+    return fc.tuple(fc.array(arb), arb).map(([sample, src]): Candid<T[]> => {
+        const equals = (a: T[], b: T[]) =>
+            arraysAreEqual(a, b, sample[0]?.equals ?? deepEqual);
+        return {
             value: sample.map((sample) => sample.value),
             src: {
                 candidType: `Vec(${src.src.candidType})`,
-                imports: new Set(src.src.imports).add('Vec')
-            }
-        })
-    );
+                imports: new Set(src.src.imports).add('Vec'),
```

Comment:
Let's try to not do `.add` so check for all the places you're using it. I won't comment on this again.